### DISCUSSION TITLE
Tuplets

### DISCUSSION THREAD

**Discussion by rodolfo-medina** [Audiveris/audiveris#494] [General]
Hi Hervé, hi all.

In the Handbook I can't find about tuplets, and Audiveris only shows 3-plets and 6-plets.  How to have other, arbitrary tuplets?

thanks,

rodolfo

**Comment by hbitteur** [comment]
Right now, Audiveris 5.2 supports only 3-plets and 6-plets, and provides no support for arbitrary tuplets.
Do you have convincing examples to share with us?

**Reply by rodolfo-medina** [reply]
In Chopin you have many such cases.  For examples, in Nocturne op.9 n.1 there is a 11-plet, a  22-plet, a 7-plet, a 20-plet; and in walz op.69 n.1 there is a 5-plet and a 13-plet...

**Reply by hbitteur** [reply]
And what is their precise impact?
I'm asking because the Wikipedia article (https://en.wikipedia.org/wiki/Tuplet) is a bit discouraging for the Audiveris developer (in other words: myself)

**Reply by rodolfo-medina** [reply]
Isn't the "logic" the same as 3-plets and 6-plets?  And what do you mean by their precise impact?

**Reply by rodolfo-medina** [reply]
You mean perhaps it is more complicated...

**Reply by hbitteur** [reply]
Yes, this is what I mean. Complex and not frequently used.
So, I'll postpone this for a future release.

**Reply by rodolfo-medina** [reply]
ok, thank you...

**Comment by rodolfo-medina** [comment]
Im'attaching Beethoven's Appassionata: on page 4, last measure, there are 5-plets, and a red background in Audiveris.  How to handle this?

Thanks,

Rodolfo

[beeth-appassionata.pdf](https://github.com/Audiveris/audiveris/files/8308848/beeth-appassionata.pdf)
